src/main/steps/common/task-list/components/notification-banner/definitions.ts

```diff
@@ -26,6 +26,7 @@ export enum NotificationType {
   CAFFCASS = 'cafcass',
   ORDER_NON_PERSONAL_SERVICE = 'orderNonPersonalService',
   ORDER_PERSONAL_SERVICE = 'orderPersonalService',
+  DA_APPLICATION_SERVED_BY_COURT_PERSONAL_NONPERSONAL_SERVICE = 'applicationServedByCourtPersonalNonPersonalServiceDA',
```

Comment:
Rename to applicationServedByCourtPersonalNonPersonalServiceToDAApplicant and pls add it below CA notificationID so that it is easy to understand when grouped